src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -372,6 +436,67 @@ g.test('formats')
     t.expectOK(result);
   });
 
+g.test('default_blend_color,initial_blend_color')
+  .desc(`Test that the initial blend color is correctly set at the beginning of every sub passes.`)
+  .fn(async t => {
+    const kFormat = 'rgba8unorm';
+    const kSize = 64;
+    const kWhiteColorData = new Float32Array([255, 255, 255, 255]);
+    const kBlackColorData = new Float32Array([0, 0, 0, 0]);
+
+    const basePipeline = t.createRenderPipelineForTest(kFormat, undefined);
+    const testPipeline = t.createRenderPipelineForTest(kFormat, {
+      srcFactor: 'constant',
+      dstFactor: 'one',
+      operation: 'add',
+    });
+
+    const renderTarget = t.device.createTexture({
+      usage: GPUTextureUsage.RENDER_ATTACHMENT | GPUTextureUsage.COPY_SRC,
+      size: [kSize, kSize],
```

Comment:
We only really need this to be 1x1. This will make the `textureContentIsOKByT2B` check much faster.

Reply:
Ah, I see. I change the value of `kSize` to `1`.